src/main/flight/imu.h

```diff
@@ -71,11 +72,13 @@ typedef struct imuRuntimeConfig_s {
     float dcm_kp;
     uint8_t acc_unarmedcal;
     uint8_t small_angle;
+    uint8_t use_gps_heading;
```

Comment:
This struct is used in a parameter group. This means that new elements can only be added at the end (any other modifications will require the version number in `PG_REGISTER_WITH_RESET_TEMPLATE` to be increased).